ant11
32

ArgumentError: activate does not accept more than one argument:
['python', 'py100Forest.py', '--numDrones', '10', '--popSize', '10', '--numGenerations', '10', '--maxRuns', '50']


no change     /root/miniconda/condabin/conda
no change     /root/miniconda/bin/conda
no change     /root/miniconda/bin/conda-env
no change     /root/miniconda/bin/activate
no change     /root/miniconda/bin/deactivate
no change     /root/miniconda/etc/profile.d/conda.sh
no change     /root/miniconda/etc/fish/conf.d/conda.fish
no change     /root/miniconda/shell/condabin/Conda.psm1
no change     /root/miniconda/shell/condabin/conda-hook.ps1
no change     /root/miniconda/lib/python3.12/site-packages/xontrib/conda.xsh
no change     /root/miniconda/etc/profile.d/conda.csh
no change     /home/fdahlke/.bashrc
No action taken.
Subscribed successfully
Trying to spawn 100 Drones
Sim not ready after 0 minutes, sleeping for 1 minute
Request to spawn drones timed out!
Drones Spawned Sucessfully
Traceback (most recent call last):
  File "py100Forest.py", line 174, in <module>
    prob_density,x,y = getProbabilityGrid(Last_Known_Position,sigma)
  File "py100Forest.py", line 99, in getProbabilityGrid
    prob_density = (1 / (np.sqrt(2 * np.pi)* sigma)) * np.exp(-((x - Last_Known_Position[0])**2 + (y - Last_Known_Position[1])**2) / (2 * sigma**2))
TypeError: 'function' object is not subscriptable
INFO:    Terminating squashfuse_ll after timeout
INFO:    Timeouts can be caused by a running background process
srun: error: ant11: task 0: Exited with exit code 1
